.github/workflows/docs-checks.yml

```diff
@@ -40,11 +40,14 @@ jobs:
         run: |
           # python -m pip install --upgrade --user pip
           pip install -r requirements/base.txt -U -f https://download.pytorch.org/whl/torch_stable.html -q
+          pip install -r requirements/extra.txt
           pip install -r requirements/docs.txt
           python --version ; pip --version ; pip list
         shell: bash
 
       - name: Test Documentation
+        env:
+          SPHINX_MOCK_REQUIREMENTS: 0
```

Comment:
this is the part that says, no moc to be used, right?

Reply:
yep, the conf.py reads the env variable and converts to bool
by default it is True, so setting it to false here manually but on RTD it will still be true